frontend/src/pages/team/changeType.vue

```diff
@@ -54,12 +47,12 @@
                             <p v-if="isContactRequired">To learn more about our {{ input.teamType?.name }} plan, click below to contact our sales team.</p>
                             <p v-if="trialMode && !trialHasEnded">Setting up billing will bring your free trial to an end</p>
                             <p v-if="!isContactRequired && team.suspended">Setting up billing will unsuspend your team</p>
-                            <p v-if="isTypeChange">Your billing subscription will be updated to reflect the new costs</p>
+                            <p v-if="!isContactRequired"> Your billing subscription will be updated to reflect the new costs</p>
                         </div>
                     </template>
                     <div class="flex gap-x-4">
                         <template v-if="!isContactRequired">
-                            <ff-button v-if="isTypeChange" :disabled="!formValid" data-action="change-team-type" @click="updateTeam()">
+                            <ff-button v-if="!billingEnabled || billingDisabledForSelectedTeamType" :disabled="!formValid" data-action="change-team-type" @click="updateTeam()">
```

Comment:
Why do we need to hide the change their team button? It basically allows users to upgrade their team type

Reply:
There are two possible cases:
 - if the team already has billing setup (or billing is not enabled for the platform at all), we show the 'Change team type' button. This will trigger the type change directly.
 - otherwise, we show the 'Setup Payment Details' button (line 58) which takes them via stripe first - and the type change only gets applied on the backend once the stripe checkout completes